test/mocks/ticker.go

```diff
@@ -0,0 +1,78 @@
+package mocks
+
+import (
+	"sync"
+	"time"
+)
+
+func NewTicker() *Ticker {
+	now := time.Now()
+
+	ticker := &Ticker{
+		sigTime:  make(chan time.Time),
+		sigClose: make(chan struct{}, 1),
+		sigAdd:   make(chan time.Duration),
+		sigReset: make(chan time.Duration),
+		d:        1000 * time.Hour,
+		ch:       make(chan time.Time, 1),
+		time:     now,
+		lastTick: now,
+	}
+
+	return ticker
+}
+
+type Ticker struct {
+	lock     sync.RWMutex
+	sigTime  chan time.Time
+	sigClose chan struct{}
+	sigReset chan time.Duration
+	sigAdd   chan time.Duration
```

Comment:
What is the purpose of that channels? I see that they are closed in `Stop()` method, but I don't anywhere that they are used by something else

Reply:
Leftover from previous implementation. Removing.